app/upgrades/v15/upgrades.go

```diff
@@ -323,3 +327,21 @@ func setBalance(
 
 	return nil
 }
+
+// SetMinInitialDepositRatio sets the MinInitialDepositRatio param of the gov
+// module to 10% - this is the proportion of the deposit value that must be paid
+// at proposal submission.
+func setMinInitialDepositRatio(ctx sdk.Context, gk govkeeper.Keeper) error {
```

Comment:
Write a small test to double check this. Check `TestUpgradeMinCommissionRate` (or any other tests, but this one already changes with params).

Reply:
See https://github.com/cosmos/gaia/pull/2913/commits/c7a7229fe62e0ec0fb7984de2119b57b8af7c406